src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$smooth-color-transition: color 300ms;
```

Comment:
I think it`s better create seperate variable for transition-duration because you can use this time for different transition-property, so it's not necessery write transition-property in variables.